products/eslint.md

```diff
@@ -0,0 +1,103 @@
+---
+title: eslint
+category: framework
+iconSlug: eslint
+permalink: /eslint
+releasePolicyLink: https://eslint.org/version-support/
+changelogTemplate: "https://github.com/eslint/eslint/releases/tag/v__LATEST__"
+releaseColumn: true
+releaseDateColumn: true
+eolColumn:  Maintenance Support
+eoasColumn: Active Support
+tags: herodevs
+customColumns:
```

Comment:
```suggestion
eoesColumn: Extended Long Term Support
customColumns:
```

The `eoesColumn` is missing. This is the standard column that documents if there is extended support (and optionally until when).